tutorial/examples/dashbio_components/molecule_3d_viewer.py

```diff
@@ -0,0 +1,46 @@
+import json
+import six.moves.urllib.request as urlreq
+
+import dash
+import dash_bio as dashbio
+import dash_html_components as html
+
+
+external_stylesheets = ['https://codepen.io/chriddyp/pen/bWLwgP.css']
+
+app = dash.Dash(__name__, external_stylesheets=external_stylesheets)
+
+model_data = urlreq.urlopen('https://raw.githubusercontent.com/plotly/dash-bio-docs-files/master/mol3d/model_data.js').read()
+styles_data = urlreq.urlopen('https://raw.githubusercontent.com/plotly/dash-bio-docs-files/master/mol3d/styles_data.js').read()
+model_data = json.loads(model_data)
+styles_data = json.loads(styles_data)
+
+app.layout = html.Div([
+    dashbio.Molecule3dViewer(
+        id='my-dashbio-molecule3d',
+        styles=styles_data,
+        modelData=model_data
+    ),
+    "Selection data",
+    html.Hr(),
+    html.Div(id='molecule3d-output')
+])
+
+
+@app.callback(
+    dash.dependencies.Output('molecule3d-output', 'children'),
+    [dash.dependencies.Input('my-dashbio-molecule3d', 'selectedAtomIds')]
+)
+def show_selected_atoms(atom_ids):
+    if atom_ids is None or len(atom_ids) == 0:
+        return 'No atoms have been selected. Click somewhere on the molecular structure to select an atom.'
```

Comment:
```suggestion
        return 'No atom selected. Click somewhere on the molecular structure to select an atom.'
```

Reply:
Could instead change to 
```suggestion
        return 'No atom(s) selected. Click somewhere on the molecular structure to select an atom.'
```
Since multiple atoms can be selected in the demo app.

Reply:
> Since multiple atoms can be selected in the demo app.

In this case, it's more important to use the plural in the second sentence: "Click on the molecular structure to select one or more atoms." (Also removing "somewhere" which doesn't sound great.)
Semantically, "no atom" means not a single one, so not one, not two, not three, etc. If you wish to say that "multiple atoms can be selected in the demo app," say it explicitly, and that belongs to the second sentence.

Reply:
https://github.com/plotly/dash-bio/pull/328#issuecomment-487315301 haha, as you prefer, but I think the parentheses are unnecessary.